bluepysnap/simulation.py

```diff
@@ -67,29 +75,34 @@ def circuit(self):
         """Access to the circuit used for the simulation."""
         from bluepysnap.circuit import Circuit
 
-        if "network" not in self.config:
-            raise BluepySnapError("No 'network' set in the simulation/global config file.")
-        return Circuit(self.config["network"])
+        if not Path(self.to_libsonata.network).is_file():
+            raise BluepySnapError(f"'network' file not found: {self.to_libsonata.network}")
+        return Circuit(self.to_libsonata.network)
+
+    @property
+    def output(self):
+        """Access the output section."""
+        return self.to_libsonata.output
 
     @property
     def run(self):
         """Access to the complete run dictionary for this simulation."""
-        return self.config["run"]
+        return self.to_libsonata.run
 
     @property
     def time_start(self):
         """Returns the starting time of the simulation."""
-        return self.run.get("tstart", 0)
+        return getattr(self.run, "tstart", 0)
```

Comment:
I don't see `tstart` in https://github.com/BlueBrain/libsonata/blob/e0cdaf707feae298d432a11742d2620721a1ae10/python/bindings.cpp#L530
or https://github.com/BlueBrain/libsonata/blob/e0cdaf707feae298d432a11742d2620721a1ae10/include/bbp/sonata/config.h#L192
and it's not even defined at https://sonata-extension.readthedocs.io/en/latest/sonata_simulation.html#run
🤔

Reply:
You're right. Me neither, didn't even check this. Just assumed this was correct in the first place. I wonder what's the correct/intended behavior here? :thinking:  

Should we even have `time_start` in the `Simulation` class to begin with? :man_shrugging:

Reply:
Open a ticket in libsonata if there is an inconsistency, pls; don't forget to look in https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md as well

Reply:
Okay. According to jdc, simulation should always start at `0` so this is not in the spec.
I'll have this return 0, then.